pymc/model.py

```diff
@@ -1259,34 +1265,6 @@ def set_data(
 
         shared_object.set_value(values)
 
-    def initial_point(self, seed=None) -> Dict[str, np.ndarray]:
```

Comment:
I think I probably caused this when trying to reorder the methods a bit more logically. There is a `shape_from_dims` below the initial point related methods which makes more sense above closed to the other coord/dims related methods.

I would therefore delete the repeated methods above and not these below.

Reply:
Alright. I will reorder

Reply:
I put after the dim/coords methods but before set_data. Thought that made enough sense